plugins/modules/monitoring/datadog/datadog_downtime.py

```diff
@@ -0,0 +1,311 @@
+---
+module: datadog_downtime
+short_description: Manages Datadog downtimes
+description:
+  - Manages downtimes within Datadog.
+  - Options as described on https://docs.datadoghq.com/api/.
+author:
+  - Datadog (@Datadog)
+requirements: [datadog-api-client]
+options:
+    api_key:
+        description:
+          - Your Datadog API key.
+        required: true
+        type: str
+    api_host:
+        description:
+          - The URL to the Datadog API.
+          - This value can also be set with the C(DATADOG_HOST) environment variable.
+        required: false
+        default: https://api.datadoghq.com
+        type: str
+    app_key:
+        description:
+          - Your Datadog app key.
+        required: true
+        type: str
+    state:
+        description:
+          - The designated state of the monitor.
+        required: true
+        choices: ["present", "absent"]
+        default: present
+        type: str
+    id:
+        description:
+          - The identifier of the downtime.
+          - If empty, a new downtime gets created, otherwise it is either updated or deleted depending of the C(state).
+          - To keep your playbook idempotent, you should save the identifier in a file and read it in a lookup.
+        type: int
+    monitor_tags:
+        description:
+          - A list of monitor tags to which the downtime applies.
+          - The resulting downtime applies to monitors that match ALL provided monitor tags.
+        type: list
+        elements: str
+    scope:
+        description:
+          - A list of scopes to which the downtime applies.
+          - The resulting downtime applies to sources that matches ALL provided scopes.
+        type: list
+        elements: str
+    monitor_id:
+        description:
+          - The ID of the monitor to mute. If not provided, the downtime applies to all monitors.
+        type: int
+    downtime_message:
+        description:
+          - A message to include with notifications for this downtime.
+          - Email notifications can be sent to specific users by using the same "@username" notation as events.
+          - Monitor message template variables can be accessed by using double square brackets, i.e "[[" and "]]".
+        type: str
+    start:
+        type: int
+        description:
+          - POSIX timestamp to start the downtime. If not provided, the downtime starts the moment it is created.
+    end:
+        type: int
+        description:
+          - POSIX timestamp to end the downtime. If not provided, the downtime is in effect until you cancel it.
+    timezone:
+        description:
+          - The timezone for the downtime.
+        type: str
+    rrule:
+        description:
+          - The C(RRULE) standard for defining recurring events.
+          - For example, to have a recurring event on the first day of each month,
+          - select a type of rrule and set the C(FREQ) to C(MONTHLY) and C(BYMONTHDAY) to C(1).
```

Comment:
```suggestion
            select a type of rrule and set the C(FREQ) to C(MONTHLY) and C(BYMONTHDAY) to C(1).
```
You split up that sentence into two paragraphs.